examples/datadog-tracing/example.rst

```diff
@@ -0,0 +1,105 @@
+.. _install_sandboxes_datadog:
+
+Datadog tracing
+===============
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+        Used to make ``HTTP`` requests.
+
+The Datadog tracing sandbox demonstrates Envoy's :ref:`request tracing <arch_overview_tracing>`
```

Comment:
lets add a `.. note::` somewhere near the top that makes it clear the need for a Datadog account - so user gets that setup before proceeding if they dont already have one.